egui/src/widgets/button.rs

```diff
@@ -55,6 +56,28 @@ impl Button {
             frame: None,
             wrap: None,
             min_size: Vec2::ZERO,
+            image: None,
+        }
+    }
+
+    #[allow(clippy::needless_pass_by_value)]
+    pub fn image_and_text(
```

Comment:
Add a docstring explaining where the image will be shown (to the left of the text?)

Reply:
Added a string above the function to describe how it works. Let me know if anything additional is needed.